src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+import { Icon, IconSize } from '../../enums/icon'
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms'
+
+export interface MultiSelectOption<T> {
+  name: string
+  value: T
+}
+
+interface SelectableOption<T> extends MultiSelectOption<T> {
+  isSelected: boolean
+}
+
+@Component({
+  selector: 'sofie-multi-select',
+  templateUrl: './multi-select.component.html',
+  styleUrls: ['./multi-select.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      multi: true,
+      useExisting: MultiSelectComponent,
+    },
+  ],
+})
+export class MultiSelectComponent<T> implements OnInit, ControlValueAccessor {
+  protected Icon = Icon
+  protected IconSize = IconSize
+
+  @Input() public options: MultiSelectOption<T>[] = []
+  public selectableOptions: SelectableOption<T>[] = []
+
+  @Input() public label: string
+  @Input() public placeholder?: string
+
+  @Output() public onChange: EventEmitter<T[]> = new EventEmitter()
+
+  public selectedOptions: SelectableOption<T>[] = []
+
+  private onChangeCallback: (value: T[]) => void
+  private onTouchedCallback: () => void
+
+  private values: T[] = []
+
+  private isTouched: boolean = false
+
+  constructor() {}
+
+  public ngOnInit(): void {
+    this.selectableOptions = this.options.map(option => {
+      return {
+        ...option,
+        isSelected: false,
+      }
+    })
+  }
```

Comment:
Changes to the options input variable, won't update the component, since this is only done on component initialization.

Reply:
We can implement that we when run into a usecase where we need to update the options.
There is a bit too many egdecases in it than I want to start implementing in this PR.